metsim/metsim.py

```diff
@@ -247,6 +249,15 @@ def met_data(self):
             self._met_data['elev'] = self.domain['elev']
             self._met_data['lat'] = self.domain['lat']
             self._met_data['lon'] = self.domain['lon']
+
+            # process constant_vars
+            template_var = list(self.params.get('forcing_vars', None).values())[0]
+            constant_vars = self.params.get('constant_vars', None)
+            for var in constant_vars.keys():
+                constant_value = float(constant_vars[var])
+                self._met_data[var] = self._met_data[template_var].copy()
```

Comment:
copying the `template_var` ensures that the constant field is of the right dimensions. The correct attributes get set when the variable is written to file

Reply:
No longer used - now using `xr.full_like()`